meta-refkit/conf/bblayers.conf.sample

```diff
@@ -20,6 +20,7 @@ REFKIT_LAYERS = " \
   ##OEROOT##/../meta-security-isafw \
   ##OEROOT##/../meta-intel-realsense \
   ##OEROOT##/../meta-clang \
+  ##OEROOT##/../meta-flatpak \
```

Comment:
@klihub based on the commit message from @pohly 8 it is because everything between 4...7 is "reserved".